xbmc/interfaces/json-rpc/schema/CMakeLists.txt

```diff
@@ -10,19 +10,8 @@ add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/ServiceDescripti
                    DEPENDS ${JSON_SRCS}
                    COMMENT "Generating ServiceDescription.h")
 
-add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/addons/xbmc.json/addon.xml
-                   COMMAND ${CMAKE_COMMAND}
-                           -DCMAKE_SOURCE_DIR=${CMAKE_SOURCE_DIR}
-                           -DCORE_BINARY_DIR=${CMAKE_BINARY_DIR}
-                           -DCORE_SYSTEM_NAME=${CORE_SYSTEM_NAME}
-                           -P ${CMAKE_CURRENT_SOURCE_DIR}/GenerateAddonXml.cmake
-                   WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-                   DEPENDS ${JSON_SRCS} ${CMAKE_SOURCE_DIR}/addons/xbmc.json/addon.xml.in
-                   COMMENT "Generating xbmc.json/addon.xml")
-
 add_custom_target(generate_json_header ALL
-                  DEPENDS ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/ServiceDescription.h
-                          ${CMAKE_BINARY_DIR}/addons/xbmc.json/addon.xml)
```

Comment:
removing this breaks auto rebuilding in case of changes to the schema.
At least in the past this was important to @Montellese

Reply:
@wsnipex can you have look at the above comment and tell if I missed anything?

Reply:
Makes sense. All good